picking_dispatch/__openerp__.py

```diff
@@ -18,7 +18,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-{'name': 'Picking dispatch', # pylint: disable=C8101
+{'name': 'Picking dispatch',  # pylint: disable=C8101
```

Comment:
What about use the standard template [__openerp__.py](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py)?
Why we need a special `__openerp__.py` here?

Could you change the disable=C8101 by the name of the check instead of number?
and add a comment of why is needed disable it